lib/src/components/NxList/__tests__/NxListItem.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import NxList from '../NxList';
+import { NxListItemProps } from '../types';
+import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+
+describe('NxListItem', function() {
+
+  const minimalProps: NxListItemProps = {};
+  const getShallow = getShallowComponent(NxList.Item, minimalProps);
+
+  it('renders the children in an .nx-list__item', function() {
+    const children = [
+      <NxList.Text key="1">Test Item 1 Text</NxList.Text>,
+      <NxList.Subtext key="2">Test Item 1 Subtext</NxList.Subtext>
+    ];
+    const contentEl = getShallow({children});
+    expect(contentEl).toExist();
+    contentEl.find('li').forEach((e) => {
```

Comment:
Why `forEach`, there should only be one

Reply:
> Why `forEach`, there should only be one

`forEach` has been removed. 
55b92e6395a57fc06d88def0b018b36a0a874305